lib/dal/src/attribute/value.rs

```diff
@@ -670,6 +672,11 @@ impl AttributeValue {
                                         .value,
                                 ]
                             }
+                            ValueSource::Secret(secret_id) => {
+                                vec![Secret::get_serialized_key_for_id(ctx, secret_id)
+                                    .await
+                                    .map_err(Box::new)?]
+                            }
```

Comment:
When executing from prototype function for direct, dependent `AttributeValues` of the secret, we will use the key and not the id.

Reply:
This is outdated